xformers/triton/dropout.py

```diff
@@ -25,42 +25,48 @@
 class _dropout(torch.autograd.Function):
     @staticmethod
     @custom_fwd(cast_inputs=torch.float16)
-    def forward(ctx, x, p, bias, activation, activation_grad):
+    def forward(ctx, x, p, bias, activation, activation_grad, trainable_bias):
```

Comment:
the trainable bias (or not) was not properly handled before (bias was always assumed to be trainable, which is mostly true but not always)